packages/server/lib/cloud/api.ts

```diff
@@ -651,6 +670,10 @@ module.exports = {
       if (!verified) {
         debugProtocol(`Unable to verify protocol signature %s`, url)
 
+        if (env.get('CYPRESS_INTERNAL_ENV') === 'test') {
```

Comment:
What is this doing?

Reply:
ah thanks for the catch, I was trying to figure out why the protocol stub was failing signature verification during system tests